prometheus_client/exposition.py

```diff
@@ -70,10 +71,18 @@ def generate_latest(registry=core.REGISTRY):
 
 class MetricsHandler(BaseHTTPRequestHandler):
     def do_GET(self):
+        try:
+            content = generate_latest(core.REGISTRY)
```

Comment:
Prometheus would handle it the same. If we're going to explicitly return a 500 we should also include the full exception in the response.